frodo.js

```diff
@@ -1,39 +1,44 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!